components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -395,10 +401,11 @@ func runPatchCommand(cmd *cobra.Command, args []string) error {
 				InputFiles:               []string{configFile},
 				ErrorCheckEnableInOutput: true,
 				NodeType:                 configCmdFlags.chef_server,
+				InputFilesPrefix:         chefserverPrefix,
 			},
 		}
-
-		postgresqlCmd := fmt.Sprintf(BACKEND_COMMAND, dateFormat, "postgresql", "%s", "postgresql"+"_"+timestamp+"_"+configFile)
+		postgresqlPrefix := "postgresql" + "_" + timestamp + "_"
```

Comment:
What happens if the user patches the config with the **same file name**?
If that is working properly, and we don't need to keep track of all the configs that are patched previously, I am good with this approach. Else consider adding a timestamp too.